system/logger/wrapper/polywrap.yaml

```diff
@@ -0,0 +1,9 @@
+format: 0.0.1-prealpha.9
+name: Logger wrapper
+language: wasm/assemblyscript
+schema: ./src/schema.graphql
+module: ./src/index.ts
+deploy: ./polywrap.deploy.yaml
+import_redirects:
+  - uri: wrap://ens/logger.core.polywrap.eth
+    schema: ../../../../monorepo/packages/core-interfaces/logger/build/schema.graphql
```

Comment:
No, it's alright since this will make updating and maintaining wrappers easier. 

If we use IPFS here and we want to update something in the interface and hence the wrapper, one needs to first publish the Interface before being able to update the wrapper, this will complicate the workflow. In the above case interface and implementation schemas will always be in sync.